pkg/encoding/simple8b/encoding.go

```diff
@@ -663,14 +660,13 @@ func pack20(src []uint64) uint64 {
 		src[15]<<45 |
 		src[16]<<48 |
 		src[17]<<51 |
-		src[18]<<54 |
-		src[19]<<57
+		src[18]<<54
 }
 
 // pack15 packs 15 values from in using 3 bits each
 func pack15(src []uint64) uint64 {
-	_ = src[14] // eliminate multiple bounds checks
-	return 5<<60 |
+	return src[14]<<56 | // eliminate multiple bounds checks
+		5<<60 |
```

Comment:
Could you instead write it at:

```go
return 5<<60 |
    src[14] << 56 | // eliminate multiple bounds checks
    ...
    ...
```

And then repeat for all the functions? I feel like having the constant on the return line is clearer.
```